main.js

```diff
@@ -77,20 +77,20 @@ const log = report => {
 	process.exit(report.errorCount === 0 ? 0 : 1);
 };
 
+const files = (report, predicate) => report.results
+	.filter(predicate)
```

Comment:
> For 0, I think we can assume the counts are always numbers

:+1: I just wasn't sure about it.

> Should the warnings only be opened if there are no errors?

Both make sense, but I've no strong preference. Feel free to decide it.

> I would prefer using the descending sorter, that way the file with the most errors will be opened first.

Mhm, I'm using sublime: all files are opend, and the last file gets the focus. What editor do you use?